docker/docs/Dockerfile

```
@@ -10,20 +10,31 @@ FROM ${PARENT_IMAGE}:${IMAGE_VERSION} AS pecandocs
 RUN apt-get update \
     && apt-get install -y --no-install-recommends pandoc \
     && Rscript \
-        -e 'repos <- c(getOption("repos"),' \
-        -e '           sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
+        -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
         -e 'remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
         -e 'remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
         -e 'remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
+        -e 'install.packages("pkgdown")' \
     && rm -rf /var/lib/apt/lists/*
 
+# ----------------------------------------------------------------------
+# Compile bookdown documentation
+# ----------------------------------------------------------------------    
 WORKDIR /src/book_source/
 COPY book_source /src/book_source/
 COPY documentation /src/documentation/
 COPY docker-compose.yml /src/
 COPY docker/env.example /src/docker/
 RUN make build
 
+# ----------------------------------------------------------------------
+# Build pkgdown documentation
+# ----------------------------------------------------------------------
+WORKDIR /src/
+COPY package_documentation /src/package_documentation
+COPY Makefile /src/
+RUN make pkgdocs
```

Because `make pkgdocs` runs in the package source directories, need to copy a few more pieces before building here:
```suggestion
WORKDIR /pkgdoc/
COPY Makefile Makefile.depends /pkgdoc/
COPY scripts/build_pkgdown.R /pkgdoc/scripts/build_pkgdown.R
COPY base /pkgdoc/base/
COPY modules /pkgdoc/modules/
COPY models /pkgdoc/models/
RUN make clean && make pkgdocs
```

---

BTW, If you want to test-build this locally you can use an incantation similar to
```
docker build \                               
    --platform linux/amd64 \
    --secret id=github_token,env=GITHUB_PAT \
    --tag "pecandocstest" \
    --build-arg PARENT_IMAGE="pecan/base" \
    --build-arg IMAGE_VERSION="develop" \
    --build-arg PECAN_VERSION="develop" \
    --file docker/docs/Dockerfile \
    .
```